contracts/dollar/hardhat.config.ts

```diff
@@ -80,20 +80,11 @@ const config: HardhatUserConfig = {
     usdcWhaleAddress: "0x72A53cDBBcc1b9efa39c834A540550e23463AAcB",
     pickleControllerAddr: "0x6847259b2B3A4c17e7c43C54409810aF48bA5210",
   },
-
-  /*   paths: {
-    deploy: "./scripts/deployment",
-    deployments: "./deployments",
-    sources: "./contracts",
-    tests: "./tests",
-    cache: "./cache",
-    artifacts: "./artifacts",
-  }, */
   networks: {
     localhost: {
       url: "http://0.0.0.0:8545",
       forking: {
-        url: `https://eth-mainnet.alchemyapi.io/v2/${process.env.API_KEY_ALCHEMY || ""}`,
+        url: process.env.MAINNET_PROVIDER_URL || `https://eth-mainnet.alchemyapi.io/v2/${process.env.API_KEY_ALCHEMY || ""}`,
```

Comment:
So the reason I made this is to make the app working with other providers as well. For example, sometimes infura, ankr is better than alchemy. but I can't use it because it just works with alchemy key which might not be extensible. Thats the reason I made this enhancement.

Reply:
I feel like we should note all the environment variables we are using in our project in our README and then eventually try and handle them automatically in our build process like we did with the default keys. Perhaps you can add a note in the README and we can get around to addressing it later? 

It would be cool to maybe do a Promise.race and choose the best performer lol maybe one day.

Reply:
> 

Promise.race might be more useful for some microservices. but if we can get it, it will be awesome.